project-base/storefront/hooks/cart/useAddOrderItemsToCart.ts

```diff
@@ -0,0 +1,37 @@
+import { useCurrentCart } from 'connectors/cart/Cart';
+import { useAddOrderItemsToCartMutationApi } from 'graphql/generated';
+import { getInternationalizedStaticUrls } from 'helpers/localization/getInternationalizedStaticUrls';
+import { useDomainConfig } from 'hooks/useDomainConfig';
+import { useRouter } from 'next/router';
+import { Dispatch, SetStateAction, useState } from 'react';
+
+export const useAddOrderItemsToCart = (): [
+    string | undefined,
+    Dispatch<SetStateAction<string | undefined>>,
+    (orderUuid: string) => void,
+    (orderUuid: string, shouldMerge: boolean) => void,
+] => {
+    const [orderUuid, setOrderUuid] = useState<string>();
+    const { cart, isCartEmpty } = useCurrentCart();
+    const router = useRouter();
+    const { url } = useDomainConfig();
+    const [cartUrl] = getInternationalizedStaticUrls(['/cart'], url);
+    const [, AddOrderItemsToCart] = useAddOrderItemsToCartMutationApi();
```

Comment:
probably typo with capital letter?

```suggestion
    const [, addOrderItemsToCart] = useAddOrderItemsToCartMutationApi();
```

Reply:
Right, done